Simplenote/src/main/res/values/dimens.xml

```diff
@@ -44,6 +44,7 @@
     <dimen name="corner_radius">4dp</dimen>
     <dimen name="divider_height">1dp</dimen>
     <dimen name="elevation_bar">8dp</dimen>
+    <dimen name="navigation_elevation">16dp</dimen>
```

Comment:
```suggestion
    <dimen name="navigation_elevation">16dp</dimen>
    <dimen name="navigation_margin">22dp</dimen>
```

The `navigation_margin` dimension could be used for the navigation drawer margin mentioned above.